content/features/search-filters.md

```diff
@@ -0,0 +1,40 @@
++++
+title = "Searching on Crunch (just got better)"
+description = "With these three new search filters, you can refine your queries and find the datasets or variables you are looking for faster."
+date = "2017-07-10T12:59:58-04:00"
+weight = 20
+draft = false
+tags = ["search"]
+categories = ["feature"]
++++
+
+Search has been a part of Crunch from the beginning. In a system designed to house and share large volumes of data, the ability to quickly find the data that's important right now is critical. We're continually working to improve how data is indexed to make searching faster and more accurate, and we just added a couple new ways for you to refine your searches to get to what you are looking for faster and more easily.
+
+## Search Filters
+
+Two dropdown menus at the top of the panel help you to dial in your search.
+
+
+{{< figure src="../images/SearchTypeFilter.png" class="floating-left">}}
+
+On the left, the **Type filter** lets you specify **Datasets**, **Variables**, or **Categories** to limit your search to that type of object. For example, imagine you had a dataset titled "Changing attitudes towards Gender Identity"; you might try searching for it using the term **Gender**, only to find that you get dozens of results from every dataset with a "Gender" variable. By limiting the search to **Datasets**, you can quickly find what you are looking for.
+
+<p style="clear:both;"></p>
+
+{{< figure src="../images/SearchDateFilter.png" class="floating-right">}}
+
+On the right, the **Date filter** lets you limit your search to recent datasets.
+
+
+Select a time period to limit your search to that time. We use the **End Date** property of the dataset to determine the date of a dataset. This property can be viewed in [Dataset Properties](http://support.crunch.io/crunch/crunch_dataset-properties.html), and is typically set by the data owner when the dataset is imported.

```

Comment:
"Select a time period to limit your search to that time" seems a little obvious. I suggest rephrasing it, or perhaps just deleting it entirely--maybe the sentence above is sufficient.